src/styles/components/_component-grid.scss

```diff
@@ -69,6 +179,10 @@ $breakpoint-tablet: 820px;
       &--max-scott {
         grid-area: max-scott;
       }
+
+      border: 1px solid;
+      text-align: center;
+      align-items: center;
```

Comment:
These are actually taken care of elsewhere, so you could delete these too I think.

Reply:
Yep, they were not needed so they are deleted now!